Fonts/493470/753D1281059CB826E.css

```diff
@@ -16,4 +16,4 @@
?#hco') format('embedded-opentype'); font-style:normal; font-weight:400; } @font-face { font-family:"Knockout 94 4r"; src: url('//apprentices.seesparkbox.com/Fonts/493470/E9C18E7A68DBB6040.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/E9C18E7A68DBB6040.eot?#hco') format('embedded-opentype'); font-style:normal; font-weight:400; } @font-face { font-family:"Knockout 26 A"; src: url('//apprentices.seesparkbox.com/Fonts/493470/7FCB0935E93E0FBE0.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/7FCB0935E93E0FBE0.eot?#hco') format('embedded-opentype'); font-style:normal; font-weight:400; } @font-face { font-family:"Knockout 26 4r"; src: url('//apprentices.seesparkbox.com/Fonts/493470/7FCB0935E93E0FBE0.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/7FCB0935E93E0FBE0.eot?#hco') format('embedded-opentype'); font-style:normal; font-weight:400; } @font-face { font-family:"Archer A"; src: url('//apprentices.seesparkbox.com/Fonts/493470/85845EA0864BB5730.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/85845EA0864BB5730.eot?#hco') format('embedded-opentype'); font-style:normal; font-weight:500; } @font-face { font-family:"Archer 5r"; src: url('//apprentices.seesparkbox.com/Fonts/493470/85845EA0864BB5730.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/85845EA0864BB5730.eot?#hco') format('embedded-opentype'); font-style:normal; font-weight:500; } @font-face { font-family:"Archer A"; src: url('//apprentices.seesparkbox.com/Fonts/493470/2E2221D311E64A3CA.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/2E2221D311E64A3CA.eot?#hco') format('embedded-opentype'); font-style:italic; font-weight:500; } @font-face { font-family:"Archer 5i"; src: url('//apprentices.seesparkbox.com/Fonts/493470/2E2221D311E64A3CA.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/2E2221D311E64A3CA.eot?#hco') format('embedded-opentype'); font-style:italic; font-weight:500; } @font-face { font-family:"Archer A"; src: url('//apprentices.seesparkbox.com/Fonts/493470/C1763FE6CCDAC4FA7.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/C1763FE6CCDAC4FA7.eot?#hco') format('embedded-opentype'); font-style:normal; font-weight:800; } @font-face { font-family:"Archer 8r"; src: url('//apprentices.seesparkbox.com/Fonts/493470/C1763FE6CCDAC4FA7.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/C1763FE6CCDAC4FA7.eot?#hco') format('embedded-opentype'); font-style:normal; font-weight:800; } @font-face { font-family:"Archer A"; src: url('//apprentices.seesparkbox.com/Fonts/493470/7FE003FA659397C49.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/7FE003FA659397C49.eot?#hco') format('embedded-opentype'); font-style:italic; font-weight:800; } @font-face { font-family:"Archer 8i"; src: url('//apprentices.seesparkbox.com/Fonts/493470/7FE003FA659397C49.eot'); src: url('//apprentices.seesparkbox.com/Fonts/493470/7FE003FA659397C49.eot?#hco') format('embedded-opentype'); font-style:italic; font-weight:800; } 
```

Comment:
To make this more robust/portable, we could actually just use a relative path only and remove the domain altogether.